2.0/m_apacheauth.cpp

```diff
@@ -0,0 +1,159 @@
+/*       +------------------------------------+
```

Comment:
this header style changed a while ago. it'd be nice if you could use the newer style (and your own copyright info)